features/conversation/conversation-chat/conversation-attachment/conversation-message-multi-remote-attachments.tsx

```
@@ -53,44 +57,46 @@ type ISingleAttachmentDisplayProps = {
 }
 
 const SingleAttachmentDisplay = memo(function SingleAttachmentDisplay({
-  attachment: attachmentMetadata,
+  attachment: attachment,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)